app/views/course/announcements/_announcement.html.slim

```diff
@@ -0,0 +1,20 @@
+div class=(announcement.currently_valid? ? '' : 'future')
```

Comment:
Yes, but I'll not change the color, I will only change color for future
announcements which student cannot see.

On Thu, Jan 22, 2015 at 11:44 AM, Joel Low <notifications@github.com> wrote:

> In app/views/course/announcements/_announcement.html.slim
> <https://github.com/Coursemology/coursemology2/pull/62#discussion_r23354003>
> :
>
> > @@ -0,0 +1,20 @@
> > +div class=(announcement.currently_valid? ? '' : 'future')
>
> expired announcements still display, according to your controller
> implementation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/62/files#r23354003>.
>